Makefile

```
@@ -37,10 +37,11 @@ recette_rebase_onto_production: ## Deployment: Rebase the recette branch and its
 	git checkout recette && git rebase --rebase-merges production
 
 recette_merge_in_production:    ## Deployment: Fast-forward the production branch to the recette branch
+	git checkout recette && git tag `date +%Y-%m-%d-%H-%M-%s`-deploy-recette-to-production
```

Sur ma machine, 
```
> date +%H-%M-%s
15-36-1641479782
> date +%H-%M-%S
15-36-25
```
Je crois que %s, c’est les secondes depuis 1970.

```suggestion
	git checkout recette && git tag `date +%Y-%m-%d-%H-%M-%S`-deploy-recette-to-production
```